framework/src/media/media_recorder.c

```diff
@@ -194,8 +194,10 @@ record_result_t media_stop_record(void)
 
 	g_rc.state = RECORD_STOPPING;
 	RECORD_UNLOCK();
+	
+	pthread_join(g_rc.pth, NULL);
 
-	pthread_join(g_rc.pth, NULL);	
+	printf("media_stop_record done.\n");
```

Comment:
Actually it is framework, not application so please get rid of printf.
We should refine some debug command instead of printf.